tests/tests.py

```diff
@@ -45,6 +61,12 @@ def test_first_page(self):
         self.assertTrue(page.has_next)
         self.assertFalse(page.has_previous)
 
+    async def test_async_first_page(self):
+        page = await self.paginator.apage(first=2)
+        self.assertSequenceEqual(page, [self.items[0], self.items[1]])
+        self.assertTrue(page.has_next)
+        self.assertFalse(page.has_previous)
```

Comment:
etc etc for the rest of the tests

Reply:
Admittedly I have not run any tests in this repo yet... 👀

Reply:
let's add a test for last too 😊